frontend/src/libs/components/link/link.tsx

```diff
@@ -2,14 +2,14 @@ import { NavLink } from 'react-router-dom';
 
 import { type AppRoute } from '#libs/enums/enums.js';
 import { getValidClassNames } from '#libs/helpers/helpers.js';
-import { type ValueOf } from '#libs/types/types.js';
+import { type ClassValue, type ValueOf } from '#libs/types/types.js';
 
 import styles from './styles.module.scss';
 
 type Properties = {
   to: ValueOf<typeof AppRoute>;
   children: React.ReactNode;
-  className?: string;
+  className?: ClassValue;
```

Comment:
we shouldn't pass className here, if we need to add margins we should wrap component where it's used, if we need additional styles, we should pass specific properties
```suggestion
```

Reply:
I've noticed that the className property is also used in the Link component in other places, not connected to this task (sidebar, meditation entry). If I remove this prop, other links will be with broken styles. Should I try to refactor links in other places?

Reply:
ok, then let's leave it, I will create another task for this